tensorboard/webapp/metrics/store/metrics_reducers_test.ts

```diff
@@ -2820,21 +2821,6 @@ describe('metrics reducers', () => {
         });
       });
 
-      it('keeps linkedTimeEnabled disabled if previously disabled', () => {
```

Comment:
This should still pass right? I guess it is a weird test. I think it was added because of a bug.